packages/keyring/src/pair/index.ts

```diff
@@ -6,7 +6,7 @@ import { Keypair, KeypairType } from '@polkadot/util-crypto/types';
 import { KeyringPair, KeyringPair$Json, KeyringPair$Meta, SignOptions } from '../types';
 import { PairInfo } from './types';
 
-import { u8aConcat, assert } from '@polkadot/util';
+import { u8aConcat } from '@polkadot/util';
```

Comment:
```suggestion
import { assert, u8aConcat } from '@polkadot/util';
```

Includes `assert` used below.

Reply:
Done